public/Template/assets/mobirise/css/mbr-additional.css

```diff
@@ -148,17 +148,17 @@ body {
 }
 .btn-primary,
 .btn-primary:active {
-  background-color: #149dcc !important;
-  border-color: #149dcc !important;
+  background-color: yellowgreen !important;
+  border-color: yellow !important;
   color: #ffffff !important;
 }
 .btn-primary:hover,
 .btn-primary:focus,
 .btn-primary.focus,
 .btn-primary.active {
   color: #ffffff !important;
-  background-color: #0d6786 !important;
-  border-color: #0d6786 !important;
+  background-color: limegreen !important;
```

Comment:
Please return this the way it is. There is a colored theme that is being used across the whole project. And the same for the border-color below.